src/js/pages/Escrow/components/CancelEscrow.jsx

```diff
@@ -1,22 +1,32 @@
+/* eslint-disable no-alert */
+/* eslint-disable no-restricted-globals */
+
 import React from 'react';
 import {Row, Col} from 'reactstrap';
 import PropTypes from 'prop-types';
 import {faTimes} from "@fortawesome/free-solid-svg-icons";
 
 import RoundedIcon from "../../../ui/RoundedIcon";
+import escrow from '../../../features/escrow';
+
 
-const CancelEscrow = () => (
-  <Row className="mt-4 text-primary">
-    <Col xs="2">
-      <RoundedIcon icon={faTimes} bgColor="blue"/>
-    </Col>
-    <Col xs="10" className="my-auto">
-      <h6 className="m-0">Cancel Trade</h6>
-    </Col>
-  </Row>
-);
+const CancelEscrow = ({cancelEscrow, trade}) => {
+  const shouldDisplay = trade.status === escrow.helpers.tradeStates.waiting || trade.status === escrow.helpers.tradeStates.funded;
+  return shouldDisplay && <a onClick={() => { if(confirm('Sure?')) cancelEscrow(trade.escrowId); }}>
```

Comment:
Good idea! I'll create a PR for replacing all the confirms by something more easy to look at! (specially since I see on the figma desing that the confirm dialogs sometimes have titles, which is not something the standard `confirm` supports).